mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowHttpCaller.java

```diff
@@ -1,5 +1,7 @@
 package org.mlflow.tracking;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Importing this to mark

```
  MlflowHttpCaller(MlflowHostCredsProvider hostCredsProvider,
                   int maxRateLimitIntervalMs,
                   int rateLimitRetrySleepInitMs,
                   int maxRetryAttempts,
                   HttpClient client) {
```

as visible for testing purposes (it's only used by `HttpCallerTest`)